.github/actions/build-go/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+function build() {
+  echo "========================= Building pktvisor-cli ========================="
+  cp -rf golang/ /src/
+  cp -rf ./version.go /src/pkg/client/version.go
```

Comment:
please document here that this depends on the cpp build step having been run first to produce version.go